<?php
return [
    'link' => '链接',
    'links' => '网址',
    'all_links' => '所有链接',
    'recent_links' => '最近的链接',
    'no_links' => '无链接',

    'add' => '添加链接',
    'add_quick' => '快速添加链接',
    'show' => '显示链接',
    'details' => '链接详情',
    'edit' => '编辑链接',
    'update' => '更新链接',
    'delete' => '删除链接',

    'private' => '私人链接',

    'history' => '历史',
    'history_added' => '添加 <code>:newvalue</code> 到 :fieldname',
    'history_changed' => ':fieldname 从 <code>:oldvalue</code> 更改为 <code>:newvalue</code>',
    'history_removed' => '从 :fieldname 中删除 <code>:oldvalue</code>。',
    'history_deleted' => '链接已删除',
    'history_restored' => '链接已恢复',
    'history_created' => '链接已创建',

    'url' => '链接',
    'title' => '网站标题',
    'description' => '描述',
    'revtags' => '标签',
    'revlists' => '列表',
    'is_private' => '隐私状态',

    'status' => [
        '1' => '运行中',
        '2' => '移动',
        '3' => '已失效',
    ],

    'author' => '创建者：:user',

    'external_link' => '外部链接',
    'wayback' => '链接存档 @ Wayback Machine',

    'check_disable' => '禁用网站检查',
    'check_disabled' => '已禁用检查',
    'check_enable' => '启用网站检查',
    'check_enabled' => '已启用检查',

    'added_successfully' => '已添加链接。',
    'added_connection_error' => '链接已添加，但在尝试访问 URL 时发生连接错误。详细信息可以在日志中找到。',
    'added_request_error' => '链接已添加，但在尝试请求URL时发生错误，例如无效的证书。 详细信息可以在日志中找到。',
    'updated_successfully' => '已更新链接。',
    'deleted_successfully' => '已删除链接。',
    'deletion_error' => '无法删除链接。',

    'duplicates_found' => 'LinkAce发现提交的 URL可能重复：',

    'notifications.linkcheck.errors' => 'LinkAce在检查您的链接时发生错误。',
    'notifications.linkcheck.errors.moved' => '⚠ 以下链接移动到一个新位置:',
    'notifications.linkcheck.errors.broken' => '🚫 以下链接不再可访问或返回一个错误:',

    'happy_bookmarking' => '快乐的收藏书签把！',
];
